extensions/ql-vscode/src/view/model-editor/ModelEditor.tsx

```diff
@@ -148,6 +148,10 @@ export function ModelEditor({
             setInProgressMethods(new Set(msg.methods));
             break;
           }
+          case "setSentToLLMMethods": {
+            // TODO: set state
```

Comment:
We'll need to add this as React state and pass it down through the UI elements to where it's needed, but this can be done in a future PR.